practise1.js

```diff
@@ -0,0 +1,123 @@
+﻿var practise1 = (
+	function () {
+		"use strict";
+
+		var _curry = function (func, args) {
```

Comment:
Carrying algorithm is too complex. Each call returns new function. Carry function could be wrapped in another one. Args can be used in closure.